MantidQt/CustomInterfaces/src/Tomography/TomographyIfacePresenter.cpp

```diff
@@ -19,12 +19,28 @@
 #include <QThread>
 #include <QTimer>
 
+#include "MantidQtCustomInterfaces/Tomography/TomoToolConfigDialogBase.h"
+
 using namespace Mantid::API;
 using namespace MantidQt::CustomInterfaces;
 
 namespace MantidQt {
 namespace CustomInterfaces {
 
+const std::string TomographyIfacePresenter::g_defOutPathLocal =
+#ifdef _WIN32
+    "D:/imat-data/";
```

Comment:
This is the default path for the IMAT pc, options to change directory are available in the settings